### PR TITLE
My Portifolio Personal

### PR DESCRIPTION
**Pull request by iankiema**
The following files have been added for this project:
-Index.html
-Style.css
-Images
-README.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+        </main>
+   </header>
+
+   <!-- Recent Work Section-->
+   <section class="work-container">
+
+        <div class="project-title">
+            <div class="header-title">
+                <h1>My Recent Works</h1>
```
Comment (binhussen) [inline index.html]: - Kindly you will have one `h1` element per page, so use other heading elements for this, I think `h2` would be better for better semantic arrangement 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h1>Profesional Art Printing Data</h1>
```
Comment (binhussen) [inline index.html]: - Kindly you will have one `h1` element per page, so use other heading elements for this for better semantic arrangement 👍

File: index.html [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
```
Comment (binhussen) [inline index.html]: - Kindly you will have one `h1` element per page, so use other heading elements for this for better semantic arrangement 👍

File: index.html [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
```
Comment (binhussen) [inline index.html]: - Kindly you will have one `h1` element per page, so use other heading elements for this for better semantic arrangement 👍

File: index.html [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
```
Comment (binhussen) [inline index.html]: - Kindly you will have one `h1` element per page, so use other heading elements for this for better semantic arrangement 👍

File: index.html [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+            </div>
+        </div>
+
+   </section>
+
+   <!--About Me Section-->
+
+   <section class="about-me">
+       <div class="my-details">
+          <h1>About me</h1>
```
Comment (binhussen) [inline index.html]: - Kindly you will have one `h1` element per page, so use other heading elements for this, I think `h2` would be better for better semantic arrangement 👍

File: index.html [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+            </div>
+
+            <div class="main-content">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul>
+                    <li>CSS</li>
+                    <li>HTML</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
```
Comment (binhussen) [inline index.html:77]: - Kindly, would arrange the right margin of your recent framework list to stick to the Figma design and have a better UI 👍  
Screenshot 2023-06-05 123338

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @iankiema , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Well-structured code ✔️
- Sticked the Given Design ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 8]
```diff
@@ -0,0 +1,268 @@
...
+            </div>
+        </div>
+
+   </section>
+
+   <!--About Me Section-->
+
+   <section class="about-me">
+       <div class="my-details">
+          <h1>About me</h1>
```
Comment (Shisui6) [inline index.html]: As the previous reviewer suggested, please update this element to use another `<hx>` tag such as `<h2>` to improve the semantics of your webpage🙏

File: index.html [hunk 9]
```diff
@@ -0,0 +1,268 @@
...
+
+        </main>
+   </header>
+
+   <!-- Recent Work Section-->
+   <section class="work-container">
+
+        <div class="project-title">
+            <div class="header-title">
+                <h1>My Recent Works</h1>
```
Comment (Shisui6) [inline index.html]: As the previous reviewer suggested, please update this element to use another `<hx>` tag such as `<h2>` to improve the semantics of your webpage🙏

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @iankiema, 

# Requires Changes 🍀
Great work on making the changes requested by a previous reviewer 👍
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet🪲.


 ## Highlights 🎯
 - Good use of flexbox ✅
 - Good design ✅
 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 8 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 9 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @iankiema 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.


## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**